torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -68,13 +68,26 @@ def _to_bf16(input_call):
             _users=1,
         )
 
-    def _unary_fusion_pattern(unary_fusion, call_fn, users, is_bf16):
-        # only insert to_dtype if is_bf16 is True
+    def _to_fp16(input_call):
+        return CallFunction(
+            prims.convert_element_type.default,
+            input_call,
+            KeywordArg("to_fp16"),
+            _users=1,
+        )
+
+    def _unary_fusion_pattern(unary_fusion, call_fn, users, lp_dtype):
+        # only insert to_dtype if lp_dtype is True
         computation_call = (
-            _to_float(call_fn(), users=users) if is_bf16 else call_fn(users=users)
+            _to_float(call_fn(), users=users) if lp_dtype else call_fn(users=users)
```

Comment:
Perhaps `lowp_dtype` is more readable?

Reply:
Thanks for advice, changed `lp` to `lowp`